.github/ISSUE_TEMPLATE/bug_template.md

```diff
@@ -8,13 +8,13 @@ labels: "type : bug"
 ## Issue
 
 Describe the issue you are facing. Show us the implementation: screenshots, gif, etc.
- 
+
 ## Expected
 
 Describe what should be the correct behaviour.
- 
+
 ## Steps to reproduce
 
-1. 
-2. 
-3. 
+1.
+2.
+3.
```

Comment:
The spaces are here on purpose: So that you can just start typing your text instead of starting with a space 💭 
(I remember I wanted to remove them a long time ago xD)

Reply:
Make sense, I will bring it back 🤞

Reply:
Bring this back and also adjust the workflow naming in https://github.com/nimblehq/rails-templates/pull/388/commits/e80dbff82801f36f5b5aa1501c0b82264b4b7bf7